_Boards/Atmel/Board_Mega/MFBoards.h

```diff
@@ -44,9 +59,11 @@
 #define MOBIFLIGHT_NAME         "MobiFlight Mega"
 #define EEPROM_SIZE             4096    // EEPROMSizeMega
 #define MEMLEN_CONFIG           1496    // max. size for config which wil be stored in EEPROM
-#define MEMLEN_CONFIG_BUFFER    1000    // max. size for configBuffer, contains only names from inputs
+#define MEMLEN_NAMES_BUFFER    1000    // max. size for configBuffer, contains only names from inputs
```

Comment:
Why did we change this?

Reply:
The buffer is no longer for config info altogether, but specifically for names only, so it became misleading (especially associated with the now unrelated MEMLEN_CONFIG)